cmd/ears/subcmd/run.go

```diff
@@ -80,6 +80,7 @@ var runCmd = &cobra.Command{
 			fx.Invoke(syncerfx.SetupDeltaSyncer),
 			fx.Invoke(tablemgr.SetupRoutingManager),
 			fx.Invoke(quotamanagerfx.SetupQuotaManager),
+			fx.Invoke(app.SetupOpenTelemetry),
 			fx.Invoke(app.SetupAPIServer),
```

Comment:
I think we should put SetupAPIServer last, so that we setup everything else first before we serve up the API

Reply:
i tried that, it didn't work - i think because setup api server initializes the global logger (that's probably wrong...) which is needed by sharder

Reply:
I see. That is fine then. Can we log a bug about it, so we can fix it later